src/queries/playlists.ts

```diff
@@ -33,3 +36,39 @@ export const fetchCollectorPlaylists = async (
     .filter(playlists => playlists.type === 'custom')
     .map(parseApiPlaylist);
 };
+
+export const createPlaylist = async (
+  playlist: IPlaylist,
+  signer: Signer,
+): Promise<IPlaylist> => {
```

Comment:
`IPlaylist` type is a little limited here because I assumed that SDK will work only on `custom` type playlists. 
So maybe you need to add `type: 'custom'` here to the payload?

Reply:
yeh - we could add follows to the sdk but i would name it IFollow or something like that rather than IPlaylist

Reply:
So follows and favs should be exposed through different functions?

Reply:
maybe we don't need follows for now in SDK?

Reply:
OK I restricted everything to 'custom'

Reply:
yeh def follows/favs in different functions if we expose them